docker/pulumi/Dockerfile

```diff
@@ -63,6 +63,10 @@ RUN curl -fsSLo /tmp/go.tgz https://golang.org/dl/go${GOLANG_VERSION}.linux-amd6
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 
+# Install Java
+RUN apt-get update -y && \
+  apt-get install -y \
+  maven
```

Comment:
For readability, suggest either indenting this b/c it's a param of the previous line, or merging into the previous line.